test/specs/homePage.js

```diff
@@ -0,0 +1,15 @@
+describe('crossfadeYT Music Video', function(){
+  it('should click on crossfade logo to return home page', function(){
+    const crossHomePage = require('./crossHomePage'); //importing Page Objects
+
+    const crossfadeYT = new crossHomePage();
+
+    crossfadeYT.open();
+    crossfadeYT.logo_page();
+
+    browser.url('http://localhost:3000/');
```

Comment:
this is not correct. This test says, `Go to home page, click on logo, go to http://localhost:3000/, assert we are on homepage`

We don't want to call `browser.url()` here. We want to click the logo and assert, because the test case is checking the button action